app/models/git/track.rb

```diff
@@ -49,6 +64,18 @@ def snippet_filepath
       "docs/SNIPPET.txt"
     end
 
+    def debug_filepath
+      "exercises/shared/.docs/debug.md"
+    end
+
+    def help_filepath
+      "exercises/shared/.docs/help.md"
+    end
+
+    def tests_filepath
+      "exercises/shared/.docs/tests.md"
+    end
+
```

Comment:
I'd probably use some metaprogramming here too.

I'd probably define a FILEPATHS constant with hash of `{help: "exercises/shared/.docs/help.md", ...}` and then generate these methods from that.

That keeps all the "data" in one structure and makes it easier/clearer to see/update